.github/workflows/ci.yml

```diff
@@ -54,12 +54,24 @@ jobs:
        - name: Start stripe-mock
          run: docker run -d -p 12111-12112:12111-12112 stripe/stripe-mock && sleep 5
        - name: Test
-         run: make test &&
-              make bench &&
-              make check-api-clients
+         run: make ci-test
        - name: Coveralls
          run: make coverage && make coveralls
          if: matrix.go == '1.16'
          env:
            COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
            COVERALLS_FLAG_NAME: Go-${{ matrix.go }}
+
+  publish:
```

Comment:
Added publish target so we get a release notification even though there isn't actually any package push happening.